### PR TITLE
M2

### PR DESCRIPTION
**Pull request by devwistkey**

### CODE REVIEW

File: components/ZilPay/zilpay-base.ts [hunk 1]
```diff
@@ -149,7 +149,7 @@ export class ZilPayBase {
             let xInit = '0x2d7e1a96ac0592cd1ac2c58aa1662de6fe71c5b9'
 
             if (net === 'testnet') {
-                XWALLET = 'zil1qvyhwc7s3g5eayrkqf88qens9qgzxqwtd6nsje'
+                XWALLET = 'zil1nrxvlzmt4vyd5z2r3pklcg78hutez7kenq7lr3'
```
Comment (tralkan) [inline components/ZilPay/zilpay-base.ts:152]: Hi @devwistkey, why this change & the tiktok icons?

File: components/Header/index.tsx [hunk 2]
```diff
@@ -206,6 +208,7 @@ function Header() {
 
     return (
         <>
+            {showZilpay && <ZilPay />}
```
Comment (tralkan) [inline components/Header/index.tsx:211]: Why do we need this?

File: components/SSI/WalletInfo/index.tsx [hunk 3]
```diff
@@ -201,7 +202,20 @@ function Component(props: InputType) {
                                 color: isLight ? '#000' : '#dbe4eb',
                             }}
                         >
-                            {loadingInfoBal ? <Spinner /> : infoBal} {currency}
+                            {loadingInfoBal ? (
+                                <Spinner />
+                            ) : infoBal === null &&
+                              currency.toLowerCase() === 'zil' ? (
+                                <div
+                                    onClick={() => updateShowZilpay(true)}
```
Comment (tralkan) [inline components/SSI/WalletInfo/index.tsx:210]: the unlock zilpay button is throwing a toast only once but not opening the zilpay window

**Review by tralkan — CHANGES_REQUESTED** [review]

File: components/ZilPay/zilpay-base.ts (hunk 1 above)
Comment (devwistkey) [reply components/ZilPay/zilpay-base.ts:152]: Noted, I'll try to revert the xwallet. Maybe that happened when checkout branch @tralkan

File: components/Header/index.tsx (hunk 2 above)
Comment (devwistkey) [reply components/Header/index.tsx:211]: Hi, I moved the zilpay component to the header component so we can use it globally especially when Modal is shown @tralkan

File: components/SSI/WalletInfo/index.tsx (hunk 3 above)
Comment (devwistkey) [reply components/SSI/WalletInfo/index.tsx:210]: This usually happen when you are alredy unlock and lock more than one time at one session

**Review by tralkan — APPROVED** [review]
